cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -0,0 +1,277 @@
+                moveXBlockListView, parentToChildMap, categoryMap, createChildXBlockInfo,
+                verifyBreadcrumbViewInfo, verifyListViewInfo, getDisplayedInfo, clickForwardButton,
+                clickBreadcrumbButton, verifyXBlockInfo, nextCategory;
+
+            parentToChildMap = {
+                course: 'section',
+                section: 'subsection',
+                subsection: 'unit',
+                unit: 'component'
+            };
+
+            categoryMap = {
+                section: 'chapter',
+                subsection: 'sequential',
+                unit: 'vertical',
+                component: 'component'
+            };
+
+            beforeEach(function() {
+                setFixtures(
+                    "<div class='breadcrumb-container'></div><div class='xblock-list-container'></div>"
+                );
+                TemplateHelpers.installTemplates([
+                    'move-xblock-list',
+                    'move-xblock-breadcrumb'
+                ]);
+            });
+
+            afterEach(function() {
+                moveXBlockBreadcrumbView.remove();
+                moveXBlockListView.remove();
+            });
+
+            createChildXBlockInfo = function(category, options, xblockIndex) {
+                var cInfo =
+                    {
+                        category: categoryMap[category],
+                        display_name: category + '_display_name_' + xblockIndex,
+                        id: category + '_ID'
+                    };
+
+                return createXBlockInfo(parentToChildMap[category], options, cInfo);
+            };
+
+            createXBlockInfo = function(category, options, outline) {
+                var cInfo =
+                    {
+                        category: categoryMap[category],
+                        display_name: category,
+                        children: []
+                    },
+                    xblocks;
+
+                xblocks = options[category];
+                if (!xblocks) {
+                    return outline;
+                }
+
+                outline.child_info = cInfo; // eslint-disable-line no-param-reassign
+                _.each(_.range(xblocks), function(xblockIndex) {
+                    cInfo.children.push(
+                        createChildXBlockInfo(category, options, xblockIndex)
+                    );
+                });
+                return outline;
+            };
+
+            createCourseOutline = function(options) {
+                var courseOutline = {
+                    category: 'course',
+                    display_name: 'Demo Course',
+                    id: 'COURSE_ID_101'
+                };
+
+                return createXBlockInfo('section', options, courseOutline);
+            };
+
+            renderViews = function(courseOutlineJson, ancestorInfo) {
+                moveXBlockBreadcrumbView = new MoveXBlockBreadcrumbView({});
```

Comment:
In my next PR I have removed these views, `MoveModal. renderViews` would be responsible to handle these views